Jenkinsfile

```diff
@@ -763,6 +793,11 @@ class CompilerBuildEnvironment extends BuildEnvironment {
 	String getTag() { "compiler-$name" }
 	@Override
 	boolean requiresDefaultBuildArtifacts() { false }
+
+	@Override
+	boolean generatesCoverage() {
+		return false
+	}
```

Comment:
Well... Arguably the JDK 17 run is important as it tests additional record-related stuff. Not sure if this results in more coverage though.

Reply:
Isn't `compiler` just about Eclipse? and it probably doesn't bring much since it just tries to compile things 🙈

Reply:
Ah, sorry, I misread the diff:

![image](https://github.com/hibernate/hibernate-search/assets/412878/adf256d4-60c3-4934-8770-c8cf5d34ebe6)

You're right, resolving.